### DISCUSSION TITLE
raspberrypi

### DISCUSSION THREAD

**Discussion by furensic** [MrBrax/LiveStreamDVR#62] [General]
Hey would this work on a raspberry pi 3 recording 2-3 streams simultaneously?

**Comment by MrBrax** [comment]
probably not impossible with some external storage media

**Reply by furensic** [reply]
I was thinking of mounting a Google team drive and then saving on to there

**Reply by MrBrax** [reply]
You could always try i guess, but it sounds like that could create some problems unless you have an extremely good link to google's servers, if they even allow streaming to files just like that
